src/Plugin/Drop/Handler.php

```diff
@@ -0,0 +1,56 @@
+<?php declare(strict_types=1);
+
+/*
+  Copyright (c) 2024, Manticore Software LTD (https://manticoresearch.com)
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License version 2 or any later
+  version. You should have received a copy of the GPL license along with this
+  program; if you did not, you can find it at http://www.gnu.org/
+*/
+
+namespace Manticoresearch\Buddy\Base\Plugin\Drop;
+
+use Manticoresearch\Buddy\Core\Error\GenericError;
+use Manticoresearch\Buddy\Core\ManticoreSearch\Client;
+use Manticoresearch\Buddy\Core\Plugin\BaseHandlerWithClient;
+use Manticoresearch\Buddy\Core\Task\Task;
+use Manticoresearch\Buddy\Core\Task\TaskResult;
+
+final class Handler extends BaseHandlerWithClient
+{
+
+	/**
+	 * Initialize the executor
+	 *
+	 * @param Payload $payload
+	 * @return void
+	 */
+	public function __construct(public Payload $payload) {
+	}
+
+	/**
+	 * Process the request
+	 *
+	 * @return Task
+	 * @throws GenericError
+	 */
+	public function run(): Task {
+		$taskFn = static function (Payload $payload, Client $client): TaskResult {
+
+			$stmt = "DROP TABLE {$payload->table}";
+			$result = $client->sendRequest($stmt);
+			if ($result->hasError()) {
+				throw GenericError::create(
+					"Can't drop table {$payload->table}" .
+					'Reason: ' . $result->getError()
```

Comment:
It's a bit weird when we use different quotes. I think it's better to use the same approach as in the first part: `"Reason: {$result->getError()}"`. This will help us be more consistent in how we interpret variables and expressions.